### DISCUSSION TITLE
Add 512kb VRAM support to the PS/1 2121 on-board VGA chip

### DISCUSSION THREAD

**Discussion by Fenix770** [86Box/86Box#4205] [Ideas]
I found in this site https://ps1stuff.wordpress.com/documentation/ibm-ps1-model-2121/#SVGA, that the IBM PS/1 2121 on-board VGA chip support 512kb of RAM. The video chip is similar to the included on the  PS/2 Model 25SX  machine.

**Comment by Cacodemon345** [comment]
This is already emulated I think.

**Reply by TC1995** [reply]
As far as I know, we still use the biosless PS/1 VGA chip emulation on 2121, which has 256kb of memory.
